ui/pages/settings/info-tab/info-tab.test.tsx

```diff
@@ -29,9 +29,10 @@ describe('InfoTab', () => {
 
     it('should have correct href for "Attributions" link', () => {
       const attributionsLink = getByText('Attributions');
+      const metamaskVersion = process.env.METAMASK_VERSION;
```

Comment:
Is this set to `undefined`? We may want to explicitly set `process.env.METAMASK_VERSION` to a test value, and check that the test value is set.

The `process.env` values we use within the app are not all available within a unit test environment. Some are hard-coded in `test/env.js`

Reply:
Thank you for bringing this to my attention. It was indeed was resolving to undefined, and adding `METAMASK_VERSION` to `test/env.js` make more sense for the test.